app.js

```diff
@@ -1,20 +1,63 @@
 const express = require('express');
-const logger = require('morgan');
 const cors = require('cors');
-const contactsRouter = require('./routes/api/contacts');
-const { v4: uuidv4 } = require('uuid');
+const mongoose = require('mongoose');
+const passport = require('passport');
+
+require('dotenv').config();
 
 const app = express();
+const PORT = process.env.PORT || 3001;
+const uriDb = process.env.DB_URI;
 
-app.use(logger('dev'));
-app.use(cors());
 app.use(express.json());
+app.use(cors());
+app.use(passport.initialize());
+app.use(express.static('public'));
+
+const avatarRouter = require('./routes/avatarRoutes');
+const contactsRouter = require('./routes/contactRoutes');
+const usersRouter = require('./routes/userRoutes');
+const registerRouter = require('./routes/registerRoutes');
+const loginRouter = require('./routes/loginRoutes');
+const logoutRouter = require('./routes/logoutRoutes');
 
-app.use('/api', contactsRouter);
+app.use('/api/contacts', contactsRouter);
+app.use('/users', usersRouter);
+app.use('/users', registerRouter);
+app.use('/users', loginRouter);
+app.use('/users', logoutRouter);
+app.use('/users', avatarRouter); 
+
+app.use((req, res) => {
+  res.status(404).json({
+    status: 'error',
+    code: 404,
+    message: 'Not found',
+    data: 'Not found',
+  });
+});
 
 app.use((err, req, res, next) => {
   console.error(err.stack);
-  res.status(500).json({ message: err.message });
+  res.status(400).json({
+    status: 'fail',
+    code: 400,
+    message: err.message,
+    data: 'Bad request',
+  });
 });
 
-module.exports = app;
+mongoose
```

Comment:
Na przyszłość unikaj w pull request robienia rzeczy nie związanych z zadaniem. Jeśli masz jakiś inne rzeczy do zrobienia to zrób w osobnym.